indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -115,15 +117,27 @@ class Calendar extends React.Component {
 
     renderExtraButtons = () => {
         const {showUnused} = this.state;
+        const {filters: {myBookings}, actions: {setFilterParameter}} = this.props;
+
         return (
-            <Popup trigger={<Button size="large"
-                                    primary={!showUnused}
-                                    icon={showUnused ? 'minus square outline' : 'plus square outline'}
-                                    onClick={this.toggleShowUnused} />}>
-                {showUnused
-                    ? <Translate>Hide unused rooms</Translate>
-                    : <Translate>Show unused rooms</Translate>}
-            </Popup>
+            <>
+                <Popup trigger={<Button size="large"
+                                        primary={!showUnused}
+                                        icon={showUnused ? 'minus square outline' : 'plus square outline'}
+                                        onClick={this.toggleShowUnused} />}>
+                    {showUnused
+                        ? <Translate>Hide unused rooms</Translate>
+                        : <Translate>Show unused rooms</Translate>}
+                </Popup>
+                <Popup trigger={<Button size="large"
+                                        primary={myBookings}
+                                        icon="user circle"
+                                        onClick={() => setFilterParameter('myBookings', !myBookings || null)} />}>
+                    <Translate>
```

Comment:
Can nicely fit in one line

Reply:
Yeah, but this is really subjective ;P I like it much more like this.